Components/Csharp/CMakeLists.txt

```diff
@@ -40,10 +40,6 @@ if(OGRE_BUILD_COMPONENT_OVERLAY)
 endif()
 
 if(OGRE_BUILD_COMPONENT_BITES)
-	if(SDL2_FOUND)
-		include_directories(${SDL2_INCLUDE_DIRS})
```

Comment:
do you know if this gets build in any/all of the CI builds? If so I could also rely on the transitive target for Java below. If not it might be broken ...

Reply:
the MSVC and Ubuntu Builders have all bindings enabled